gammapy/datasets/tests/test_datasets.py

```diff
@@ -88,10 +89,15 @@ def test_datasets_info_table():
     table = datasets_hess.info_table()
     assert table["name"][0] == "23523"
     assert table["name"][1] == "23526"
+    assert np.isnan(table["npred_signal"][0])
 
-    table = datasets_hess.info_table(cumulative=True)
-    assert table["name"][0] == "stacked"
-    assert table["name"][1] == "stacked"
+    table_cumul = datasets_hess.info_table(cumulative=True)
```

Comment:
Maybe one should add numerical tests for the cumulative and non-cumulative case, such that the mentioned issue will never happen (the excess is here, but maybe also the counts and the background as there are the root values to compute the excess).
One should maybe test the mask usage... Be sure that the behaviour is correct without mask and with mask